examples/commoncrawl/src/main/entity-config/databases/final-database.json

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
We want to gitignore src/main/entity-config; everything is code-generated in this directory